test/cmd/newapp.sh

```diff
@@ -106,22 +148,6 @@ os::cmd::expect_failure 'oc get dc/mysql'
 os::cmd::expect_failure 'oc get dc/php'
 os::cmd::expect_success_and_text 'oc new-app -f test/testdata/template-without-app-label.json -o yaml' 'app: ruby-helloworld-sample'
 
-# check reuse imagestreams
-os::cmd::expect_success "oc new-build -D $'FROM node:8\nRUN echo \"Test\"' --name=node8"
-os::cmd::try_until_success 'oc get imagestreamtags node:8'
-os::cmd::expect_success "oc new-build -D $'FROM node:10\nRUN echo \"Test\"' --name=node10"
-os::cmd::try_until_success 'oc get imagestreamtags node:10'
```

Comment:
per my comment above, i think these 4 lines should be retained/restored.  I don't think this behavior is covered/tested in what you have above.

Reply:
restored in next push